docs/modules/ROOT/pages/index.adoc

```diff
@@ -7,3 +7,12 @@ chart] on a Syn-enabled cluster.
 The component defaults to provisioning a Bitnami Postgres database via the
 Keycloak helm chart, but this behavior can be overridden using the
 `keycloak.postgres` parameters exposed by the component.
+
+[NOTE]
+====
+This component currently doesn't support vanilla Kubernetes versions older than 1.14.
```

Comment:
```suggestion
This component doesn't support Kubernetes versions older than 1.14 anymore.
```

We do not need to support older K8s versions.

Reply:
done=